cnd/src/seed.rs

```diff
@@ -12,82 +12,109 @@ use std::{
 };
 use thiserror;
 
-pub const SEED_LENGTH: usize = 32;
+const SEED_LENGTH: usize = 32;
+pub type Seed = [u8; SEED_LENGTH];
```

Comment:
I am not sure if this only being a type-alias helps much?

I'd either go for a real newtype (so you can attach functionality to it), or not having the type alias.

Reply:
Done.